.vscode/settings.json

```diff
@@ -9,11 +9,46 @@
   "[html]": {
     "editor.formatOnSave": false
   },
+  "[ejs]": {
+    "editor.formatOnSave": false
+  },
+  "[julia]": {
+    "editor.formatOnSave": false,
+    "editor.wordBasedSuggestions": "off",
+    "editor.defaultFormatter": "julialang.language-julia",
+    "editor.wordSeparators": "`~#$%^&*()-=+[{]}\\|;:'\",.<>/?"
+  },
+  "[python]": {
+    "diffEditor.ignoreTrimWhitespace": false,
+    "gitlens.codeLens.symbolScopes": ["!Module"],
+    "editor.formatOnSave": false,
+    "editor.formatOnType": false,
+    "editor.wordBasedSuggestions": "off",
+    "editor.defaultFormatter": "ms-python.black-formatter"
```

Comment:
I found we could use Black formatter for Python from VSCODE doc: https://code.visualstudio.com/docs/python/formatting#_choose-a-formatter

The extension is this one: 
https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter

Again if we do this, next time we'll update some .py file, there will be some style change. So if we choose to do this, I can't make a PR only with style change to anticipate

Reply:
It seems Ruff could be a good option too: https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff

Community based, but it seems a wide used formatter now (see #python discussion on slack)

Reply:
You might also want to add/use `isort` to organise Python imports which linter such as PyLint will flag:
- https://marketplace.visualstudio.com/items?itemName=ms-python.isort

Reply:
FYI, when working with Python, I am using: Pylance, isort, and Black Formatter.